diesel_compile_tests/tests/fail/derive/multiple_table_names.stderr

```diff
@@ -0,0 +1,23 @@
+error: expected a single table name attribute
+  --> tests/fail/derive/multiple_table_names.rs:29:43
+   |
+29 | #[diesel(table_name = users, table_name = users_)]
+   |                                           ^^^^^^
```

Comment:
It would be even better, if the error would be reported in this way:
```suggestion
error: expected a single table name attribute
  --> tests/fail/derive/multiple_table_names.rs:29:43
   |
29 | #[diesel(table_name = users, table_name = users_)]
   |                              ^^^^^^^^^^^^^^^^^^^ note: remove this attribute
```

Reply:
agree. I updated the code to check the number of `table_name` attributes when building `Model`, instead of when retrieving them from `Model`. Now, the error will point to the `table_name` attribute, with a friendly note. 

Btw I'm not sure how to put the note right after the highlighting carets as you provided, using `proc_macro_error`. Would be nice if you had any advice on this.

Reply:
That's currently not possible, as the rust-lang proc macro crate  does not provide API's for that. See https://github.com/rust-lang/rust/issues/54140 for details.